### DISCUSSION TITLE
Alignment length and query sequence length units

### DISCUSSION THREAD

**Discussion by iwilkie** [bbuchfink/diamond#531] [Q&A] [answered]
I just wanted to ask about the units of "alignment length" and "query sequence length" from the diamond blastx tabular output. 

I've tried looking into the manual, but haven't seen anything which specifies what the units are (nucleotides or amino acids) or which specifies if the two values have the same unit.

Thank you in advance!

**Comment by bbuchfink** [comment, marked as answer]
Alignment length is in amino acids, query length is in amino acids in blastp mode and in bases in blastx mode.

**Reply by iwilkie** [reply]
Thank you so much, I appreciate the quick reply!
